.openpublishing.redirection.json

```diff
@@ -1,6 +1,41 @@
 { 
   "redirections": [ 
     {   
+        "source_path": "aspnetcore/client-side/cdn.md", 
+        "redirect_url": "/aspnet/core//client-side/", 
```

Comment:
@tdykstra I guess ur moving stuff around; however, I wonder about the double-slash here ... intentional?